core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -337,6 +342,32 @@ protected Integer getLivenessCheckPort() {
         }
     }
 
+    /**
+     * @return the port on which to check if the container is ready
+     */
+    protected List<Integer> getLivenessCheckPorts() {
```

Comment:
since the implementation uses Set as a type of collection, why don't we return Set here as well? Would make more sense, also List indicates the order

Reply:
👍